hw/dv/sv/tl_agent/dv/env/tl_agent.core

```diff
@@ -0,0 +1,26 @@
+CAPI=2:
```

Comment:
Perhaps rename this file to `tl_agent_env.core` to avoid confusion with `tl_agent/tl_agent.core`